olive/model/__init__.py

```diff
@@ -475,7 +475,7 @@ def __init__(
         model_loader: Union[str, Callable] = None,
         model_script: Union[str, Path] = None,
         script_dir: Union[str, Path] = None,
-        io_config: Union[Dict[str, Any], IOConfig] = None,
+        io_config: Union[Dict[str, Any], IOConfig, str] = None,
```

Comment:
do we need support Callable like line 479?

Reply:
Sounds good.
Also for this: https://github.com/microsoft/Olive/blob/main/olive/model/hf_utils.py#L25

How about to implement it with separated PR?

Reply:
Please add TODO